cli/oyster-cvm/Readme.md

```diff
@@ -0,0 +1,50 @@
+# Enclave Image Builder CLI
+
+A CLI tool to build and deploy your apps to the oyster marketplace. Using just your Docker configurations.
+
+## Prerequisites
+
+- Docker
+- Nix
+- Git (for fetching flakes)
+
+## Installation
+
+From source:
+
+```bash
+git clone https://github.com/yourorg/enclave-image-builder.git
+cd enclave-image-builder
+cargo build --release
```

Comment:
more setup details required before this. It's not clear that where is this repo coming from .

Reply:
updated the repo to monorepo. Will also add the prebuilt binary uri for for the cli post deployment.